test/Microsoft.AspNetCore.Mvc.FunctionalTests/ErrorPageTests.cs

```diff
@@ -69,5 +64,33 @@ public ErrorPageTests(MvcTestFixture<ErrorPageMiddlewareWebSite.Startup> fixture
             Assert.Contains("/Views/ErrorFromViewImports/_ViewImports.cshtml", content);
             Assert.Contains(expectedMessage, content);
         }
+
+        [Fact]
+        public async Task RuntimeErrorAreListedByErrorPageMiddleware()
+        {
+            // The desktop CLR does not correctly read the stack trace from portable PDBs. However generating full pdbs
+            // is only supported on machines with CLSID_CorSymWriter available. On desktop, we'll skip this test on 
+            // machines without this component.
```

Comment:
I'm not sure how to repro this. I tried this Win7 \ Win 10 machines without VS and it had the SymWriter component installed (does Office install it?). I'll add a reference to DiaSymReader just as well. If that's the case, can we specifically return false if it's not Mono?